.solcover.js

```diff
@@ -12,4 +12,12 @@ module.exports = {
         grep: "@skip-on-coverage", // Find everything with this tag
         invert: true, // Run the grep's inverse set.
     },
+    // Optimization Settings
+    // configureYulOptimizer: true,
+    // solcOptimizerDetails: {
+    //     yul: true,
+    //     yulDetails: {
+    //         optimizerSteps: ""
+    //     },
+    // },
```

Comment:
nit: looks like left-over commented code that can be removed.

Reply:
I believe there's a bug in the solcover package that requires this.

Reply:
Comment removed in favour of writing it to `SOLIDITY_SETTINGS` in `.env.sample` with a descriptive comment 👍🏾